src/main/java/org/osc/sdk/controller/api/SdnRedirectionApi.java

```diff
@@ -33,211 +33,230 @@
+     * Deleted an inspection hook
      *
-     * @param inspectionHookId
-     *            The identifier of the inspection hook to be removed.
+     * @param inspectionHookId  the identifier of the inspection hook
+     * @throws Exception upon failure
      */
     void removeInspectionHook(String inspectionHookId) throws Exception;
 
     /**
-     * Retrieves a previously installed inspection hook given its identifier.
+     * Retrieves an InspectionHookElement
      *
-     * @param inspectionHookId
-     *            The identifier of the inspection hook to be retrieved.
+     * @param inspectionHookId  the identifier of the inspection hook
+     * @return the inspection hook element, see {@link InspectionHookElement}
+     * @throws Exception upon failure
      */
     InspectionHookElement getInspectionHook(String inspectionHookId) throws Exception;
 
     /**
-     * Remove previously installed network hooks placed on behalf of network port.
+     * Removes all the inspection hooks
      *
-     * @param networkElem
-     *            The port on which behalf traffic is being inspected or a networkElement containing protected ports
+     * @param networkElem  provides context information of the network element, see {@link NetworkElement}
+     * @throws Exception upon failure
      */
     void removeAllInspectionHooks(NetworkElement networkElem) throws Exception;
 
     /**
-     * Set data path policy tag of a specific inspection hook.
+     * Sets the inspection hook tag
      *
-     * @param inspectedPort
-     *            The port on which behalf traffic is being inspected.
-     * @param inspectionPort
-     *            The inspection port belonging to network inspection device.
-     * @param tag
-     *            The tag value to set
+     * @param inspectedPort  provides context information of the network element, see {@link NetworkElement}
+     * @param inspectionPort  provides context information of the inspection port element, see{@link InspectionPortElement}
+     * @param tag  provides a tag
+     * @throws NetworkPortNotFoundException when port is not found
+     * @throws Exception upon failure
      */
     void setInspectionHookTag(NetworkElement inspectedPort, InspectionPortElement inspectionPort, Long tag)
             throws NetworkPortNotFoundException, Exception;
 
     /**
-     * Return the current tag used for an installed inspection hook.
+     * Retrieves the inspection hook tag
      *
-     * @param inspectedPort
-     *            The port on which behalf traffic is being inspected.
-     * @param inspectionPort
-     *            The inspection port belonging to network inspection device.
-     * @return The current inspection tag
+     * @param inspectedPort  provides context information of the network element, see {@link NetworkElement}
+     * @param inspectionPort  provides context information of the inspection port element, see{@link InspectionPortElement}
```

Comment:
I strongly recommend removing the see, look at the resulting javadoc, it already provides the link to the parameter type.